docs/quickstart.md

```diff
@@ -473,15 +402,18 @@ metadata:
   selfLink: ""
 ```
 
-Once the `Stack` is installed and configured skip to [Install Application].
+Note: Azure requires an [Azure MySQLServerVirtualNetworkRule] to be created *after* the `MySqlInstance` claim is bound as part of deploying the `Application` instance. A new [Composition feature](https://github.com/crossplane/crossplane/issues/1343) will remove this extra step. See [Azure MySQLServerVirtualNetworkRule] for details.
+
+#### Next Steps: Deploy an Application
+Once the Azure `Stack` is installed and configured skip to [Install Application].
 
 ## Install Application
 ### Install the Wordpress Application
 ```
 # optional alternate registry
 # REGISTRY=registry.upbound.io
 
-PACKAGE=crossplane/app-wordpress:v0.4.0
+PACKAGE=crossplane/app-wordpress:v0.5.0
 NAMESPACE=workspace1
```

Comment:
nitpick: maybe create this via `kubectl create namespace workspace1 || true`

Reply:
will add as a prerequisite in the `Setup Environment` section, which accommodates environments and namespaces/workspaces being created in different ways.

Reply:
fixed